src/WtsApi32.Desktop/WtsApi32+WtsConnectStateClass.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Runtime.InteropServices;
+    using System.Text;
+
+    /// <content>
+    /// Exported functions from the WtsApi32.dll Windows library
+    /// that are available to Desktop apps only.
+    /// </content>
+    public static partial class WtsApi32
+    {
+        /// <summary>
+        ///     Specifies the connection state of a Remote Desktop Services session.
+        /// </summary>
+        public enum WtsConnectStateClass
```

Comment:
I've saw in other places enum names were PascalCased, instead of using native names (see https://github.com/AArnott/pinvoke/blob/master/src/Kernel32.Desktop/Kernel32%2BCharacterAttributesFlags.cs or https://github.com/AArnott/pinvoke/blob/master/src/Kernel32.Desktop/Kernel32%2BCreateProcessFlags.cs, for example). Is this should be avoided? Maybe add this to contributing guide?